### PR TITLE
Add BlocStamp (BST) Support.

### PR DESCRIPTION
**Pull request by kuchtak**

### CODE REVIEW

File: common/defs/bitcoin/blockstamp.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
+  "coin_name": "BlockStamp",
+  "coin_shortcut": "BST",
+  "coin_label": "BlockStamp",
+  "website": "https://blockstamp.info",
+  "github": "https://github.com/BlockStamp/bst",
+  "maintainer": "The BlockStamp Developers <support@blockstamp.info>",
```
Comment (prusnak) [inline common/defs/bitcoin/blockstamp.json]: add your email; not some alias; you will be responsible for keeping the definition always up to date

File: common/defs/bitcoin/blockstamp.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    "Normal": 140,
+    "High": 200
+  },
+  "dust_limit": 546,
+  "blocktime_seconds": 60,
+  "uri_prefix": "blockstamp",
+  "min_address_length": 27,
+  "max_address_length": 34,
+  "bitcore": [
+    "https://explorer.blockstamp.info"
```
Comment (prusnak) [inline common/defs/bitcoin/blockstamp.json]: This is not a valid bitcore backend; remove this entry

File: common/defs/bitcoin/blockstamp.json (hunk 1 above)
Comment (kuchtak) [reply common/defs/bitcoin/blockstamp.json]: Thank you for comment. Already changed.

File: common/defs/bitcoin/blockstamp.json (hunk 2 above)
Comment (kuchtak) [reply common/defs/bitcoin/blockstamp.json]: Removed.

**Pull request merged**